### ISSUE TITLE
Tutorial - Client/Server - Error: "'PMonitor' does not contain a definition for 'TryRandom'"

### ISSUE THREAD

**Issue by thesammiller** [p-org/P#539]
I am new, so forgive me if I am getting something wrong. 
I have followed the installation instructions and am trying to run the first tutorial with:
`pc -proj:ClientServer.pproj`

Output:
```
----------------------------------------
==== Loading project file: ClientServer.pproj
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PSrc/AbstractBankServer.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PSrc/Server.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PSrc/Client.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PSrc/ClientServerModules.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PSpec/BankBalanceCorrect.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PTst/TestDriver.p
....... includes p file: /home/smiller/Development/csharp/P/Tutorial/1_ClientServer/PTst/Testscript.p
----------------------------------------
----------------------------------------
Parsing ...
Type checking ...
Code generation ...
Generated ClientServer.cs.
----------------------------------------
Compiling ClientServer...
Error:
Compiling generated C# code FAILED!
Microsoft (R) Build Engine version 16.7.3+2f374e28e for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
PGenerated/ClientServer.cs(178,63): error CS1061: 'PMonitor' does not contain a definition for 'TryRandom' and no accessible extension method 'TryRandom' accepting a first argument of type 'PMonitor' could be found (are you missing a using directive or an assembly reference?) [/home/smiller/Development/csharp/P/Tutorial/1_ClientServer/ClientServer.csproj]

Build FAILED.

PGenerated/ClientServer.cs(178,63): error CS1061: 'PMonitor' does not contain a definition for 'TryRandom' and no accessible extension method 'TryRandom' accepting a first argument of type 'PMonitor' could be found (are you missing a using directive or an assembly reference?) [/home/smiller/Development/csharp/P/Tutorial/1_ClientServer/ClientServer.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.76
```

I can see in `PGenerated/ClientServer.cs` there is a method signature which takes `PMonitor currentMachine` as an argument:
`        public static PrtMap CreateRandomInitialAccounts(PrtInt numAccounts, PMonitor currentMachine)`

This seems to be incorrect as there is a previous method signature which takes `PMachine currentMachine` as an argument:
`        public static PrtMap CreateRandomInitialAccounts(PrtInt numAccounts, PMachine currentMachine)`

I am not sure how to fix this issue. Can someone please advise? Thank you!

**Comment by ankushdesai** [comment]
hey @thesammiller, thanks for pointing this out. My most recent changes (rushed near the weekend) broke the tutorials. 
I have fixed the problem and we will release another version in sometime with the fix. We are adding Tutorials to our regressions as well so that such issues dont happen again.

Thanks! and please let us know if you face any other issues. Cheers!

**Comment by ankushdesai** [comment]
The latest version released 1.1.19 should fix this issue. Please let us know if you face any issues. Thanks!!

Closing the issue.
